tensorflow/tools/docker/Dockerfile.devel-gpu-cuda9-cudnn7

```diff
@@ -4,7 +4,7 @@ MAINTAINER Gunhan Gulsoy <gunan@google.com>
 
 # It is possible to override these for releases.
 ARG TF_BRANCH=master
-ARG BAZEL_VERSION=0.5.4
+ARG BAZEL_VERSION=0.6.1
```

Comment:
Is this mandatory?
I want to still keep testing with the minimum possible bazel we can use.

Reply:
Ok. I agree with you.